src/try-fetch-title.ts

```diff
@@ -3,10 +3,16 @@ import { defaultUrlHandler, getSpecialUrlHandler } from "./url-handlers";
 import cleanTitle from "./utils/clean-title";
 
 const tryFetchTitle = async (url: URL, regexes: string[][]) => {
+    if (regexes.length === 0) {
+        regexes = [[".+", "(.+)"]];
+    }
     const match = regexes.find(([pageRegex]) =>
         new RegExp(pageRegex).test(url.href),
     );
-    if (!match) return;
+    if (!match) {
```

Comment:
Should we instead make it so that if there's no match, we use the dummy regex? That would eliminate the need to configure a dummy regex in the first place.

Reply:
@jose-elias-alvarez sounds good, I've made this update. Please check it out.